buildpacks/gradle/src/gradle_command/daemon.rs

```diff
@@ -0,0 +1,47 @@
+use crate::gradle_command::GradleCommandError;
+use crate::GRADLE_TASK_NAME_HEROKU_START_DAEMON;
+use libcnb::Env;
+use libherokubuildpack::command::CommandExt;
+use std::io::{stderr, stdout};
+use std::path::Path;
+use std::process::Command;
+
+pub(crate) fn start(
+    gradle_wrapper_executable_path: &Path,
+    gradle_env: &Env,
+) -> Result<(), GradleCommandError<()>> {
+    let output = Command::new(gradle_wrapper_executable_path)
+        .args([
+            // Fixes an issue when when running under Apple Rosetta emulation
+            "-Djdk.lang.Process.launchMechanism=vfork",
```

Comment:
should this always be set or conditionally?

Reply:
Ideally we would only set it when need (i.e. running under Rosetta). However, there is no easy way to decide. AFAIK, there isn't a way to determine if we currently run under Rosetta. I think it's fine to always use the changed launch mechanism (which was the default in older OpenJDK versions anyways) and revisit in a year or so if it's still needed for local developement.